components/brave_wallet_ui/components/extension/add-suggested-token-panel/index.tsx

```diff
@@ -52,6 +58,12 @@ function AddSuggestedTokenPanel (props: Props) {
           <NetworkText>{selectedNetwork.chainName}</NetworkText>
         </TopRow>
         <Title>{getLocale('braveWalletAddSuggestedTokenTitle')}</Title>
+        <URLText>
+          <CreateSiteOrigin
+            originSpec={originInfo.originSpec}
```

Comment:
is there a reason why https://github.com/brave/brave-core/blob/60d8b0d85f0c050b78829299ef37390d48b61d10/components/brave_wallet_ui/components/shared/create-site-origin/index.tsx#L13 is checking `startsWith` rather than `===`? for instance the origin of `brave://wallet/crypto/unlock` should still be `brave://wallet`. 

`startsWith` can be bypassed if the originSpec isn't properly normalized - for instance `brave://wallet@rewards` starts with `brave://wallet` but actually resolves to `brave://rewards`.

Reply:
Good point! Updated to use `===`